DSCResources/MSFT_xKDSKey/MSFT_xKDSKey.meta.mof

```diff
@@ -0,0 +1,5 @@
+[ClassVersion("1.0.0.0"), FriendlyName("xKDSKey")]
```

Comment:
Sorry I can do that. In other modules the readme is in root. I thought it was strange one didn't exist.